openhtf/output/web_gui/src/app/station.testheader.ts

```diff
@@ -61,6 +68,17 @@ export class TestHeader implements OnChanges {
   }
 
   /**
+   * Propogates star toggle event back to HistoryHeader component.
```

Comment:
The changes were probably to accommodate the plugin I wrote to interface with Scout, but since that's an internal tool I couldn't include the plugin itself in this PR.